### ISSUE TITLE
Update Samples app to latest images

### ISSUE THREAD

**Issue by nicolejms** [radius-project/radius#7162]
### Steps to reproduce

NodeJS and OpenSSL must be updated to latest versions/packages.  

### Observed behavior

The NodeJS and OpenSSL version have known vulnerabilities and must be updated to latest. 

NodeJs (Npm) Security Update for fast-xml-parser 
NodeJs (Npm) Security Update for semver 
Alpine Linux Security Update for Open Secure Sockets Layer (OpenSSL)

### Desired behavior

All dependencies are at the latest released version. 

### rad Version

n/a

### Would you like to support us?

- [ ] Yes, I would like to support you

[AB#11150](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/11150)

**Comment by sk593** [comment]
The image getting flagged as vulnerable was a very old version of our demo app stored in ACR (we have since renamed the app and moved our samples images to GHCR). The image was deleted and is no longer getting flagged as vulnerable. I redeployed our latest samples app image to a new cluster and that has not been flagged for about a week so I'm going to close this issue for now. We can reopen if the images start getting flagged again
